### ISSUE TITLE
[Docs] Enhance dropdown menu that specifies docs version

### ISSUE THREAD

**Issue by coder12git** [meshery/meshery#9227]
### Current State
At present, there is an issue where the primary version button and its associated dropdown menu appear to be conflicting with each other, as depicted in the provided screenshot
![Screenshot 2023-10-28 131512](https://github.com/meshery/meshery/assets/108334168/3fcda596-d9f0-42ba-8d35-d38625f0d06e)
![Screenshot 2023-10-28 131526](https://github.com/meshery/meshery/assets/108334168/056612bf-e079-4581-a79e-734383b4e6c6)

### Desired State

Add adequate spacing between primary version button and its associated dropdown menu  -

Consider this as a reference -
![Screenshot 2023-10-28 131404](https://github.com/meshery/meshery/assets/108334168/6df3e537-32c9-4408-ad7b-ef5179dadc70)

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by aman44444** [comment]
@coder12git can i work on this issue ?

**Comment by coder12git** [comment]
Go ahead @aman44444 🚀

**Comment by leecalcote** [comment]
Cool!

**Comment by aman44444** [comment]
@coder12git I am unable to find the code file. Could you please provide me with a link to the file?

**Comment by coder12git** [comment]
> @coder12git I am unable to find the code file. Could you please provide me with a link to the file?

@aman44444  I apologize for the late response , check this file - https://github.com/meshery/meshery/blob/master/docs/_sass/navbar.scss

**Comment by coder12git** [comment]
@aman44444 i am hoping that you will create a PR soon as it's just one line of css :)

**Comment by aman44444** [comment]
@coder12git sorry for the late response ,I've already attempted to modify that file, but it seems that the "select" class only applies to dropdown button not to dropdown options that is why i asked for file location.

**Comment by coder12git** [comment]
> @coder12git sorry for the late response ,I've already attempted to modify that file, but it seems that the "select" class only applies to dropdown button not to dropdown options that is why i asked for file location.

Yes, i checked now, i think instead of using "select" we can use normal div thing and make our own dropdown using noral css for dropdown , in this way we will be able to achieve that thing

**Comment by aman44444** [comment]
@coder12git ok, i will try tomorrow

**Comment by coder12git** [comment]
> @coder12git ok, i will try tomorrow

@aman44444 just updated my reply which i said earlier, please check that

**Comment by coder12git** [comment]
> > @coder12git sorry for the late response ,I've already attempted to modify that file, but it seems that the "select" class only applies to dropdown button not to dropdown options that is why i asked for file location.
> 
> Yes, you are right, do one thing put that select inside a div and then style that div using "padding-bottom", i try that it works

Something like that -
<div style="padding-bottom: 5px">
<select>
</select>
</div>

**Comment by aman44444** [comment]
@coder12git we can't wrap select we have to wrap dropdown options , if your code is working make a PR

**Comment by Emmanuel-Sasere** [comment]
Is it still available?

**Comment by coder12git** [comment]
Yes @Emmanuel-Sasere , assigning you 🚀

**Comment by sandramsc** [comment]
This issue has been open for some time with no recent activity, unassigning to open it up for new contributors to give it a go.

**Comment by MonalikaPatnaik** [comment]
Hi @coder12git can I take up this issue?

**Comment by Shreyask24** [comment]
Can I work on this?

**Comment by 11happy** [comment]
Hello @sandramsc, @coder12git  I am interested to work on this issue. Can you please assign this to me.

**Comment by 11happy** [comment]
Hello , should I create a new custom dropdown? as we cant wrap the select.

**Comment by mrmahi2003** [comment]
is this issue still active @leecalcote  ? I had made some changes before a pr can be merged

**Comment by leecalcote** [comment]
@mrmahi2003, good question. Please test and confirm if this behavior remains on docs.meshery.io.

**Comment by anglerfishlyy** [comment]
![Image](https://github.com/user-attachments/assets/ede05097-897b-4f16-aef9-f21a4bac1668) It's still there , can i work on this?

**Comment by krishna4040** [comment]
@leecalcote I would like to contribute to this issue, if you could assign this to me!

**Comment by leecalcote** [comment]
@krishna4040 Please test and confirm if this behavior remains on docs.meshery.io.

**Comment by krishna4040** [comment]
@leecalcote i can still see this behaviour on the docs website there is no gap between the button and its associated drop down menu 

![Image](https://github.com/user-attachments/assets/61211db8-3afc-415c-815c-4ff643c794cd)

**Comment by RohitPeswani** [comment]
Can i work on this issue @leecalcote ?

**Comment by meher745** [comment]
Can the issue be assigned to me @leecalcote ?

**Comment by Kush172005** [comment]
Hi @leecalcote 👋

I came across this issue and would love to contribute to it. Could you please confirm if it's still open and assign it to me if possible?

**Comment by leecalcote** [comment]
Folks, thanks for volunteering. This issue is no longer valid.
